### ISSUE TITLE
Run a11y (accessibility) audit

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1068]
There should be some automated tools for this. I'm guessing a lot of it will be around alt text, aria labels, etc.

**Comment by neel-a-mehta** [comment]
cc @gcapiel

**Comment by baipai** [comment]
I did collect some of them in the [google drive](https://docs.google.com/document/d/1anrIClnBYVMjGJix8CrxhTlhMUF6oNDTsnLpe26eCRU/edit?usp=sharing), but more for the color contrast and UX/UI design part. 

But I also found out some more articles recently, they are more talking about alt text and also providing some websites to check especially for the front-end part. I put them in the second page of the same one on [google drive](https://docs.google.com/document/d/1anrIClnBYVMjGJix8CrxhTlhMUF6oNDTsnLpe26eCRU/edit?usp=sharing).

**Comment by neel-a-mehta** [comment]
Use Lighthouse from Google Chrome's web console. That gets performance, best practices, and a11y. Will post some example pdfs here

**Comment by neel-a-mehta** [comment]
Will be posting Lighthouse results here.

The admin homepage:
[Lighthouse-AdminView.pdf](https://github.com/seattle-uat/civiform/files/6461712/Lighthouse-AdminView.pdf)

**Comment by neel-a-mehta** [comment]
All right, here's the official accessibility guide from the City of Seattle: https://docs.google.com/spreadsheets/d/14ECSc4GWkw1J5j6OMyOgxDj6YjpuUyAFA0BEIEYnoeY/edit#gid=755456280.

It mostly adheres to WCAG 2.1 and Section 508, but I think it's a nice worksheet we should be filling out when possible.

**Comment by neel-a-mehta** [comment]
Google & volunteer teams have done this
